app/controllers/work_packages/activities_tab_controller.rb

```diff
@@ -213,7 +213,10 @@ def find_project
   end
 
   def find_journal
-    @journal = Journal.find(params[:id])
+    @journal = Journal
+      .with_sequence_version
+      .where(id: params[:id])
+      .first
```

Comment:
🟢 should raise `ActiveRecord::RecordNotFound` handled below
```suggestion
    @journal = Journal
      .with_sequence_version
      .find_by!(id: params[:id])
```

Reply:
I applied it sort of (used `find`) but merged it into the one commit. I'd like to keep it to a single commit so that it is easier to revert.